Resources/Prototypes/lobbyscreens.yml

```diff
@@ -9,3 +9,7 @@
 - type: lobbyBackground
   id: Room
   background: /Textures/LobbyScreens/room.png
+  
+- type: lobbyBackground
+  id: Planets
+  background: /Textures/LobbyScreens/planets.png
```

Comment:
Let's give it a slightly less generic name. I can foresee other "planet" lobby images being added in the future :P

Reply:
Changed id: planets to id: Dipper